aequilibrae/project/network/connector_creation.py

```diff
@@ -0,0 +1,124 @@
+from math import pi, sqrt
+from warnings import warn
+import numpy as np
+from scipy.cluster.vq import kmeans2, whiten
+from scipy.spatial.distance import cdist
+import shapely.wkb
+from shapely.geometry import LineString
+from aequilibrae.project.database_connection import database_connection
+from aequilibrae.project import network
+from aequilibrae import logger
+
+
+def connector_creation(geo, zone_id: int, srid: int, mode_id: str, link_types='', connectors=1):
+    if len(mode_id) > 1:
```

Comment:
There's a lot happening here, a docstring would be nice

Reply:
The docstrings are in the zone and node classes. This guy is not intended to be used independently. Does that make sense?